content/en/contributing/multi-account-region-testing.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Multiple account and region Testing"
+weight: 7
+description: >
+  How to test multi-account and multi-region compatibility.
+aliases:
+  - /developer-guide/multi-account-region-testing/
+---
+
+LocalStack community supports multiple accounts and regions compatibility. This document describes how to test your changes against cross accounts and regions.
```

Comment:
```suggestion
LocalStack supports multi accounts and regions compatibility. This document describes how to test your changes against cross accounts and regions.
```

Reply:
Since only community repo is multi accounts compatible for now, does it make sense to explicitly highlight this here @HarshCasper?

Reply:
like localstack community instead of just localstack? 🤔

Reply:
We wish to remove any pricing info, and just mention `LocalStack` as a product in itself. Hence we can remove the `community` keyword.